assets/css/global.less

```diff
@@ -1227,14 +1229,37 @@ body .guides .guides-section-white {
         display: none;
       }
     }
+    & .card-body {
+      padding: 16px 0;
+    }
+    & .img-inner {
+      position: absolute;
+      top: 50%;
+      -ms-transform: translateY(-50%);
+      transform: translateY(-50%);
+    }
     img {
+      @media (max-width: @screen-xs-min) {
+        .remove-visibility();
+      }
       max-height: 100px;
+      max-width: 100%;
+      max-height: 100%;
+      display: block;
+      margin: auto auto;
+      padding: 15px;
+      height: 85px;
```

Comment:
Max-height is duplicated here, and if you are going for a fixed height, it both max and height and width are now unnecessary. Also the height might need to be a bit bigger... Something like 1-3px more? Still seems like it's slightly off, but I would get the final thumbnail images first before working on this